Dockerfile

```diff
@@ -12,6 +12,32 @@ RUN apt-get update && apt-get install -y curl
 # Install Poetry
 RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
 
+RUN  apt-get update \
```

Comment:
If we moved these steps into the test stage of the dockerfile we could both speed up building dev/prod images, and also ultimately reduce the size of those